### ISSUE TITLE
No space before and after "/" in my language

### ISSUE THREAD

**Issue by scootergrisen** [quodlibet/quodlibet#2158]
I see the source strings for translations uses " / " with space before and after the "/".
In my language i would prefer to have no space.
I can do this for some strings in the file but i see more of this in the software like:
"Title / Version"
"Album / Disc Subtitle"

In my language i would like:
"Titel/version"
"Album/disk-undertitel"

**Comment by lazka** [comment]
That would need to be changed for all languages. Why do you need it?

**Comment by scootergrisen** [comment]
I think without space is more correct in my language and how we translate in other software in my language.
Perhaps you can add a string like "%s / %s" in the translation and i can translate that to "%s/%s" in my language and use that when you need to join strings.

**Comment by scootergrisen** [comment]
Strange the readme use different style:
`Ex Falso / Quod Libet - A Music Library/Editor/Player`
Maybe that is perfectly correct i don't know i just like to use the same style when i translate.

**Comment by declension** [comment]
Yeah I'd say we should fix that, the slashes are a bit abused IMO, or at least: confusing...

**Comment by lazka** [comment]
The ` / ` separator is used for joining tied tag previews. Their values get joined with `, `. Would `, ` make more sense?

**Comment by scootergrisen** [comment]
I don't know i haven't used the software enough to know.
